examples/inputs/eno.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: eno.azure.io/v1
+kind: Synthesizer
+metadata:
+  name: test-synth
+spec:
+  refs:
+  - key: some-input
+    resource:
+      group: someGroup
```

Comment:
Out of curiosity, does passing any string here work? And does passing an empty string work (as I'd expect for the core group)?

Reply:
It does have to be a valid group, this example just showcases that input data is passed correctly to synthesizers. I'll work on a proper example that actually reaches out to apiserver on a follow up PR.